test/parallel/test-buffer-safe-unsafe.js

```diff
@@ -7,8 +7,18 @@ const safe = Buffer.alloc(10);
 
 function isZeroFilled(buf) {
   for (let n = 0; n < buf.length; n++)
-    if (buf[n] > 0) return false;
+    if (buf[n] !== 0) return false;
```

Comment:
Also `-0` could be a factor, but I'm not sure. Any way, tests are a bit non-deterministic, since even if there is no actual zero filling, there is still a chance that some allocations would be zeros